tests/incremental/00-basic/02-changed_start_state1.patch

```diff
@@ -0,0 +1,19 @@
+--- a/tests/incremental/00-basic/02-changed_start_state1.c
++++ b/tests/incremental/00-basic/02-changed_start_state1.c
+@@ -1,13 +1,13 @@
+ #include <assert.h>
+
+-int g = 1;
++int g = 2;
+
+ int main() {
+     // After the presolve phase, g is in the start state but neither in the context nor in the start variables.
+     // If the change of the start state of main would not be propagated by the call to side on all start variables, the
+     // asserts in the incremental run would wrongly fail. Side however only joins with the previous value instead of
+     // overwriting, therefore the current imprecision.
+-    assert(g == 1);
+-    assert(g != 2);
++    assert(g != 1); // TODO (restarting)
++    assert(g == 2); // TODO
```

Comment:
What exactly is happening here? The annotations for before and after incremental run are no longer in the actual code file, but they're also in the patch? And this patch is also changing the assertions themselves, which the test originally did not do.

Reply:
- By putting the annotations for the incremental run in the patch file, the code for the parsing of the annotations can easily be reused for the incremental test case and is the same as for the usual regression tests. Also I think it is nice to separate them from the original non-incremental test case.
- The previous annotations for the incremental run of this test case were no longer verifiable (see comment below)
- Maybe I should revert the flipping of the asserts, I thought it might be clearer this way
- The `SUCCESS` annotation is superfluous, isn't it?